packages/editor-ui/src/constants.ts

```diff
@@ -39,7 +39,7 @@ export const BREAKPOINT_XL = 1920;
 
 
 // templates
-export const TEMPLATES_BASE_URL = `https://api-staging.n8n.io/`;
+export const TEMPLATES_BASE_URL = `https://api.n8n.io/`;
```

Comment:
is this used anywhere? why cannot we delete this? if this is still used by versions, let's rename it to that

Reply:
We are using in few places. For WorkflowTemplates (getWorkflowTemplates() inside api/workflows.js file) and  ValueSurvey & ContactPrompt modals.  

For the first one, maybe we can use the same thing that we are using for templates? Than I can change the constant name just for the two modals.

Reply:
Updated as per today discussion.